client/src/components/AddPresentation/index.tsx

```diff
@@ -1,11 +1,22 @@
+import { useState } from 'react'
+
 import { ReactComponent as NewPresentation } from '../../assets/PresentationIcons/addPresentation.svg'
+import { Upload } from '..'
```

Comment:
:eyes: what is this import path!

Reply:
importing from the index of the previous file 
something like `../index`